Formula/kubescape.rb

```diff
@@ -0,0 +1,37 @@
+class Kubescape < Formula
+  desc "kubescape is the first tool for testing if Kubernetes is deployed securely as defined in Kubernetes Hardening Guidance by to NSA and CISA (https://www.nsa.gov/News-Features/Feature-Stories/Article-View/Article/2716980/nsa-cisa-release-kubernetes-hardening-guidance/) "
+  homepage "https://www.armosec.io/armo-kubescape"
+  url "https://github.com/armosec/kubescape/archive/refs/tags/v1.0.88.tar.gz"
+  sha256 "5d052f1830dcab6e660ca97dade2bcf1b00410197b767060f84f17fcaf43c7cf"
+  license "Apache-2.0"
+
+  bottle do
+      sha256 cellar: :any_skip_relocation, big_sur: "8b6cdcd45abc765d1ffd8dfd5e0bedb9ae4bf93a56f69dab81c0f634ccdb2af5"
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[-s -w
+    -X github.com/armosec/kubescape/cmd.BuildNumber=v#{version.to_s}
+    -X github.com/armosec/kubescape/cautils/getter.ArmoBEURL=api.armo.cloud
+    -X github.com/armosec/kubescape/cautils/getter.ArmoERURL=report.euprod1.cyberarmorsoft.com
+    -X github.com/armosec/kubescape/cautils/getter.ArmoFEURL=portal.armo.cloud
+    ].join(" ")
+
+    system "go", "mod", "tidy"
+    system "go", "build", *std_go_args(ldflags: ldflags), "-o", "kubescape", "."
+    bin.install "kubescape"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags)
```
Doesn't this work?

Reply:
Nope doesn't seem to work.

```
==> go mod tidy
==> go build -trimpath -o=/usr/local/Cellar/kubescape/1.0.88/bin/kubescape -ldflags=-s -w -X github.com/armosec/kubescape/cmd.BuildNumber=v1.0.88 -X github.com/armosec/kubescape/cautils/getter.ArmoBEURL=api.armo.cloud -X github.com/armosec/kubescape/cautils/getter.ArmoERURL=report.euprod1.cyberarmorsoft.com -X github.com/armosec/kubescape/cautils/getter.ArmoFEURL=portal.armo.cloud -o kubescape .
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
ErrorDuringExecution: Failure while executing; `/usr/local/Cellar/kubescape/1.0.88/bin/kubescape completion bash` exited with 127. Here's the output:
1. raise
2. ignore
3. backtrace
4. irb
5. shell
```

Reply:
That seems to work too. I'll push my updates.